src/libs/ajax/ajax-override-utils.ts

```diff
@@ -79,17 +79,18 @@ export const overrideAppsWithLocalWDS = async (workspaceId?: string) => {
 
   // Get list of instances from local WDS.
   // Using base fetch here to avoid an import cycle with ajax-common.ts.
-  const instancesResponse = await fetch(`${wdsUrl}/instances/v0.2`, {
+  const instancesResponse = await fetch(`${wdsUrl}/collections/v1/${workspaceId}`, {
     headers: {
       Authorization: `Bearer ${token}`,
       'Content-Type': 'application/json',
     },
   });
-  const instanceIds: string[] = await instancesResponse.json();
+  const instances = await instancesResponse.json();
+  const instanceIds: string[] = instances.map((instance) => instance.id);
 
   // Create an instance for the given workspace if one does not already exist.
   if (!instanceIds.includes(workspaceId)) {
-    const createInstanceResponse = await fetch(`${wdsUrl}/instances/v0.2/${workspaceId}`, {
+    const createInstanceResponse = await fetch(`${wdsUrl}/collections/v1/${workspaceId}`, {
```

Comment:
this call to create-instance needs a request payload specifying the collection name/description, else it will fail. Since this is only used by `overrideAppsWithLocalWDS` I suggest the whole "Create an instance for the given workspace if one does not already exist." logic can just be removed; it's extra code that we don't really need

Reply:
Huh I think we don't need to call instance/collection APIs at all here then.